src/test/node/user.js

```diff
@@ -1,9 +1,7 @@
 import test from 'tape';
+import mockData from '../../server/config/json/mockData.js';
+import requestah from 'requestah';
```

Comment:
Here and everywhere in the PR: alphabetically sort all the import statements. 

mockData,
requestah,
test